### PR TITLE
Add attribution note

### LINKED ISSUE #9: License
Add 'If in doubt attribute' sentence to both doc here and MDN page

### PR DESCRIPTION
**Pull request by bsmth**
Fixes #9

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 MDN's content (including prose and code examples) is entirely available under various open source licenses.
 This file covers the types of content we provide and what licenses are in effect for each.
```
Comment (dipikabh) [inline LICENSE.md]:
```suggestion
This file covers the types of content we provide and the licenses that are in effect for each.
```

File: LICENSE.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
 
+If in doubt, please [add attribution to the content](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license) you're using, and link to the page on MDN where it is used.
```
Comment (dipikabh) [inline LICENSE.md]:
```suggestion
If in doubt, add [attribution](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license) to the content you're using, and add a link to the content page on MDN.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Just few nits

**Comment by bsmth** [comment]
Thanks, Dipika, I'll go ahead and merge

**Pull request merged**